packages/core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -62,6 +62,10 @@ const StyledTableRow = (c: any) =>
     })
   );
 
+const getStripedColor = (color?: string, opacity: number = 0.6) => {
```

Comment:
The same thing is happening here:

"This way of getting the background color is preferable because we get less re-renders. However, with these changes, the background color is not correct on the first render since we get undefined. I think it's because the CSS vars are not yet defined when we try to get the background value."

Reply:
We are still having some problems with this code:

https://github.com/lumada-design/hv-uikit-react/assets/43220251/2b0523e7-b5a6-48f4-9845-b24b777df756

The background color is not set on the first render and they don't update correctly.

Potential solution:

```suggestion
const [stripedColorEven, setStripedColorEven] = useState<
      string | undefined
    >();
const [stripedColorOdd, setStripedColorOdd] = useState<
      string | undefined
    >();

useEffect(() => {
      const getColor = (value?: string) => {
        if (value?.includes("var(--uikit-colors-")) {
          const color = value
            .replace("var(--uikit-colors-", "")
            .replace(")", "")
            .trim();

          return activeTheme?.colors.modes[selectedMode][color];
        } else {
          return value;
        }
};

const even = getColor(activeTheme?.table.rowStripedBackgroundColorEven);
const odd = getColor(activeTheme?.table.rowStripedBackgroundColorOdd);

setStripedColorEven(getStripedColor(even));
setStripedColorOdd(getStripedColor(odd));
}, [selectedMode, activeTheme]);

return (
      <TableRow
        ref={externalRef}
        className={cx(
          tableSectionContext.filterClassName,
          className,
          classes.root,
          classes?.[type],
          {
            [classes.hover]: hover,
            [classes.selected]: selected,
            [classes.expanded]: expanded,
            [classes.striped]: striped,
            [classes.variantList]: isList && type === "body",
            [classes.variantListHead]: isList && type === "head",
          }
        )}
        role={Component === defaultComponent ? null : "row"}
        $striped={striped}
        $stripedColorEven={stripedColorEven}
        $stripedColorOdd={stripedColorOdd}
        {...others}
      />
 );
```